scripts/spark_eval/copy_exome_to_hdfs_on_gcs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Copy exome data to HDFS on GCS
```

Comment:
When do you want to use this and when do you want to use `copy_exome_to_hdfs.sh`?

Reply:
Added some documentation to explain the difference.